src/appleseed/renderer/modeling/aov/pixeltimeaov.cpp

```diff
@@ -181,38 +184,12 @@ namespace
         void post_process_image(const Frame& frame) override
         {
             const AABB2u& crop_window = frame.get_crop_window();
+            ColorMap color_map;
 
-            // Find the maximum value.
+            float min_time = numeric_limits<float>::max();
```

Comment:
Once `find_min_max_red_channel()` properly initializes the min/max values, you will not need to initialize them here anymore.